python/ray/dashboard/client/src/UsageBar.tsx

```diff
@@ -0,0 +1,64 @@
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import createStyles from "@material-ui/core/styles/createStyles";
+import withStyles, { WithStyles } from "@material-ui/core/styles/withStyles";
+import React from "react";
+
+const blend = (
```

Comment:
Utils.tsx

Reply:
Leaving this for future refactoring if needed, as per our offline discussion.